modules/gapi/include/opencv2/gapi/garg.hpp

```diff
@@ -133,6 +133,48 @@ using GRunArgP = util::variant<
     >;
 using GRunArgsP = std::vector<GRunArgP>;
 
+//GAPI_EXPORTS cv::GRunArgsP bind(cv::GRunArgs &results);
+inline cv::GRunArgsP bind(cv::GRunArgs &results)
```

Comment:
This function should be exported, also its body should go to the `.cpp` file (whatever fits better from the `src/api` directory).

Reply:
done